addons/mail/static/tests/messaging_menu/messaging_menu.test.js

```diff
@@ -921,6 +921,160 @@ test("preview should display last needaction message preview even if there is a
+    const channelIds = pyEnv["discuss.channel"].create([
+        { name: "Channel1" },
+        { name: "Channel2" },
+        { name: "Channel3" },
+        { name: "Channel4" },
+        { name: "Channel5" },
+    ]);
+    const attachments = [
+        {
+            mimetype: "audio/mpeg",
+            name: "voicemessage",
+            icon: "fa-microphone",
+            text: "Voice Message",
+            voice_ids: [Command.create({ display_name: "voicemessage" })],
+        },
+        { mimetype: "video/mp4", name: "Video.mp4", icon: "fa-video-camera", text: "Video.mp4" },
+        { mimetype: "application/pdf", name: "File.pdf", icon: "fa-file", text: "File.pdf" },
+        { mimetype: "image/jpeg", name: "Image.jpeg", icon: "fa-picture-o", text: "Image.jpeg" },
+        { mimetype: "audio/mpeg", name: "Audio.mp3", icon: "fa-headphones", text: "Audio.mp3" },
+    ];
+
+    pyEnv["mail.message"].create(
+        attachments.map((attachment, i) => ({
+            attachment_ids: [
+                Command.create({
+                    mimetype: attachment.mimetype,
+                    name: attachment.name,
+                    res_id: channelIds[i],
+                    res_model: "discuss.channel",
+                    voice_ids: attachment.voice_ids || [],
+                }),
+            ],
+            author_id: partners[i],
+            body: "",
+            model: "discuss.channel",
+            res_id: channelIds[i],
+        }))
+    );
+    await start();
+    await click(".o_menu_systray i[aria-label='Messages']");
+    await contains(
+        ".o-mail-NotificationItem:nth-child(1) .o-mail-NotificationItem-text i.fa-microphone",
+        { count: 1 }
+    );
+    await contains(".o-mail-NotificationItem:nth-child(1) .o-mail-NotificationItem-text", {
+        text: "Partner1: Voice Message",
+    });
+    await contains(
+        ".o-mail-NotificationItem:nth-child(2) .o-mail-NotificationItem-text i.fa-video-camera",
+        { count: 1 }
+    );
+    await contains(".o-mail-NotificationItem:nth-child(2) .o-mail-NotificationItem-text", {
+        text: "Partner2: Video.mp4",
+    });
+    await contains(
+        ".o-mail-NotificationItem:nth-child(3) .o-mail-NotificationItem-text i.fa-file",
+        { count: 1 }
+    );
+    await contains(".o-mail-NotificationItem:nth-child(3) .o-mail-NotificationItem-text", {
+        text: "Partner3: File.pdf",
+    });
+    await contains(
+        ".o-mail-NotificationItem:nth-child(4) .o-mail-NotificationItem-text i.fa-picture-o",
+        { count: 1 }
+    );
+    await contains(".o-mail-NotificationItem:nth-child(4) .o-mail-NotificationItem-text", {
+        text: "Partner4: Image.jpeg",
+    });
+    await contains(
+        ".o-mail-NotificationItem:nth-child(5) .o-mail-NotificationItem-text i.fa-headphones",
+        { count: 1 }
+    );
+    await contains(".o-mail-NotificationItem:nth-child(5) .o-mail-NotificationItem-text", {
+        text: "Partner5: Audio.mp3",
+    });
```

Comment:
Can use `:eq()` rather than `:nth-child()`. This makes selector shorter.

`.o-mail-NotificationItem-text` can be removed in all selector, so each selector is much shorter (becomes a single line even).